core/src/scene/styleParam.cpp

```diff
@@ -180,6 +184,17 @@ StyleParam::Value StyleParam::parseString(StyleParamKey key, const std::string&
         }
         return vec.value;
     }
+    case StyleParamKey::text_buffer:
```

Comment:
I don't think so, not mentioned in reference issue https://github.com/tangrams/tangram-docs/issues/141.